docs/v3/core/v3.0.rst

```diff
@@ -839,7 +839,7 @@ apply to node names:
 * must not be a string composed only of period characters, e.g. ``"."`` or ``".."``
 * must not start with the reserved prefix ``"__"``
 
-To ensure consistent behaviour across different storage systems and programming
+To ensure consistent behavior across different storage systems and programming
```

Comment:
This is a UK vs US English difference, not sure it is worth worrying about.

Reply:
Heh. I tend to agree that it's not critical, but I have experienced in other projects that once the topic comes up it *keeps* coming up. We should probably just pick one and put it in the [dev guide](https://github.com/zarr-developers/governance/blob/main/CORE_DEV_GUIDE.md). (And if we can't agree then there cane be a ZEP :wink:)

Reply:
yep, for this document we should just pick one style and stick with it. I'm happy do do british english as well. the point is just to be consistent.

Reply:
Then since it's 4 to 1 in this file, @d-v-b, let's go with `behaviour` for the moment. (In zarr-python, it's 23 to 19). We will need to decide, but at least we can err on the side of fewer changes for the moment.